src/main/webapp/showVotes.html

```diff
@@ -46,8 +46,7 @@ <h5 class="card-title">Share!</h5>
           <script async defer crossorigin="anonymous"
             src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v7.0&appId=2667458256844253&autoLogAppEvents=1"
             nonce="3bSmYODx"></script>
-          <div class="fb-share-button" data-href="https://dtest" data-layout="button_count" data-size="small"><a
-              target="_blank"
+          <div class="fb-share-button" data-href="" data-layout="button_count" data-size="small"><a target="_blank"
```

Comment:
well it was a quick fix for the share button, it was only that someone add it a link in the data-href. but it need to be empty so that way it takes the current webpage